### PR TITLE
Update example.env add password note

### PR DESCRIPTION
**Pull request by aviv926**
Following the discord discussion [#](https://discord.com/channels/979116623879368755/1071165397228855327/1209286994090262528) [#](https://discord.com/channels/979116623879368755/1200197429518155907/1205215693516316682)
Add information on the correct way to change a password
You might want to change the wording of this.

### CODE REVIEW

File: docker/example.env [hunk 1]
```diff
@@ -7,6 +7,9 @@ UPLOAD_LOCATION=./library
 IMMICH_VERSION=release
 
 # Connection secret for postgres. You should change it to a random password
+# NOTE: Password must be changed before docker compose up before the first time, after that do not change password via DB_PASSWORD.
```
Comment (danieldietzler) [inline docker/example.env:10]:
```suggestion
# NOTE: This does not update the password for an already existing user.
```

**Comment by bo0tzz** [comment]
In the spirit of keeping the file as minimal as we can - is this something we need to note here?

**Comment by aviv926** [comment]
> In the spirit of keeping the file as minimal as we can - is this something we need to note here? 

I think so, it's required for those who don't know the database world closely, I think for most people this variable says "hey, you can change your database password here" that's half right and it can be confusing I think.

**Comment by aviv926** [comment]
I am thinking of sending an update for the documentation soon, it will be possible to specify the information on a new page and give a link for this page. What do you think?

**Comment by aviv926** [comment]
> Yeah, I'm not a fan of this change either. I saw just add a FAQ to the docs, that just links to the official postgres docker documentation where it clearly explains the purpose of this environment variable and its behavior.

Agreed, I believe I will send a PR soon.

**Comment by jrasm91** [comment]
> > Yeah, I'm not a fan of this change either. I saw just add a FAQ to the docs, that just links to the official postgres docker documentation where it clearly explains the purpose of this environment variable and its behavior.
> 
> Agreed, I believe I will send a PR soon.

Awesome, thanks!

**Pull request closed without merging**